src/commands/paint.ts

```diff
@@ -97,15 +97,19 @@ export function paint(
     process.stdout.write(`${chalk.bold('Snowpack')}\n\n`);
     // Dashboard
     if (devMode) {
-      process.stdout.write(
-        `  ${chalk.bold.cyan(`${devMode.protocol}//localhost:${devMode.port}`)}`,
-      );
-      for (const ip of devMode.ips) {
-        process.stdout.write(
-          `${chalk.cyan(` > `)}${chalk.bold.cyan(`${devMode.protocol}//${ip}:${devMode.port}`)}`,
-        );
+      // SERVER_START event happened
+      if (startTimeMs > 0 && port > 0 && protocol) {
+        process.stdout.write(`  ${chalk.bold.cyan(`${protocol}//localhost:${port}`)}`);
+        for (const ip of ips) {
+          process.stdout.write(
+            `${chalk.cyan(` > `)}${chalk.bold.cyan(`${protocol}//${ip}:${port}`)}`,
+          );
+        }
+        process.stdout.write('\n' + chalk.dim(`  Server started in ${startTimeMs}ms.\n\n`));
```

Comment:
After the server has started

![Screen Shot 2020-06-12 at 12 42 20 PM](https://user-images.githubusercontent.com/1369770/84536048-41686580-acaa-11ea-87fd-723a9fb4ede0.png)